spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-config-it/src/test/java/org/springframework/cloud/kubernetes/client/config/it/ConfigMapAndSecretIT.java

```diff
@@ -78,17 +74,35 @@
 
 	private static K8SUtils k8SUtils;
 
+	private static final K3sContainer K3S = new K3sContainer(DockerImageName.parse("rancher/k3s:v1.21.10-k3s1"))
+			.withFileSystemBind("/tmp/images", "/tmp/images", BindMode.READ_WRITE).withExposedPorts(80, 6443)
+			.withCommand("server") // otherwise, traefik is not installed
+			.withReuse(true);
+
 	@BeforeAll
-	public static void setup() throws Exception {
-		client = createApiClient();
+	static void setup() throws Exception {
+		K3S.start();
+		K3S.execInContainer("ctr", "i", "import", "/tmp/images/spring-cloud-kubernetes-client-config-it.tar");
+		createApiClient(K3S.getKubeConfigYaml());
 		api = new CoreV1Api();
 		appsApi = new AppsV1Api();
 		networkingApi = new NetworkingV1Api();
 		k8SUtils = new K8SUtils(api, appsApi);
+
+		RbacAuthorizationV1Api rbacApi = new RbacAuthorizationV1Api();
```

Comment:
we used to that via `run.sh`, but now, since we are moving to testcontainers, this is done in each project, also moved all the needed files for that.